kms/keysource.go

```diff
@@ -220,7 +239,7 @@ func ParseKMSContext(in interface{}) map[string]*string {
 		for _, kv := range strings.Split(in, ",") {
 			kv := strings.Split(kv, ":")
 			if len(kv) != 2 {
-				fmt.Printf("[WARNING]: KMS Encryption Context could not be parsed, context will not be used")
+				log.Printf("[WARNING]: KMS Encryption Context could not be parsed, context will not be used")
```

Comment:
nit: don't put a colon after brackets, it's redundant

Reply:
There's a different branch were I'm working on logging improvements. This branch uses logrus and log levels. Should I merge that branch into this one?